bootstrap_salt/fab_tasks.py

```diff
@@ -79,15 +79,22 @@ def install_minions():
     ec2.set_instance_tags(to_install, {'SaltMasterPrvIP': master_prv_ip})
     for inst_ip in public_ips:
         env.host_string = 'ubuntu@%s' % inst_ip
-        sudo('wget https://raw.githubusercontent.com/ministryofjustice/bootstrap-salt/master/scripts/bootstrap-salt.sh -O /tmp/moj-bootstrap.sh')
-        sudo('chmod 755 /tmp/moj-bootstrap.sh')
-        sudo('/tmp/moj-bootstrap.sh')
+        # copy the salt_utils.py from local to EC2 and chmod it
+        d = os.path.dirname(__file__)
+        saltutils = d + "/salt_utils.py"
+        if not os.path.isfile(saltutils):
+            print "ERROR: Cannot find %s" % saltutils
+            sys.exit(1)
+        put(saltutils, '/tmp')
+        sudo('cp /tmp/salt_utils.py /usr/local/bin')
```

Comment:
`put(saltutils, '/usr/local/bin/salt_utils.py`, use_sudo=True)` can be used combine these two commands into one.